lib/map/sign_up/service.rb

```diff
@@ -59,7 +59,8 @@ def agreements_body(icn, signature_name, version)
           responseDate: Time.zone.now,
           icn:,
           signatureName: signature_name,
-          version: config.agreements_version_mapping[version]
+          version: config.agreements_version_mapping[version],
+          legalDisplayVersion: config.legal_display_version
```

Comment:
did this not change specs?

Reply:
I updated the cassette to match on the request body